deploy/central/images/bc-ods-build-python.yml

```diff
@@ -0,0 +1,26 @@
+kind: BuildConfig
+apiVersion: build.openshift.io/v1
+metadata:
+  name: ods-build-python
```

Comment:
I'd align the name of the buildconfig with the name of the image being built. I think it is like that as well for Go, the BC is named `ods-go-toolset` (or at least it should :))